etc/avocado/avocado.conf

```diff
@@ -85,3 +85,7 @@ skip_broken_plugin_notification = []
 # options or test types).
 # The keyword "@DEFAULT" will be replaced with all available unused loaders.
 loaders = ['file', '@DEFAULT']
+
+[plugins.result]
+# Support for ordering result plugins *only* at this version
```

Comment:
Yep, that was the idea.  As mentioned, I'll probably lift the limitation on v1, so there may be no need to rephrase it.